### PR TITLE
Updating list of amps 

### PR DESCRIPTION
**Pull request by icodebyamanda**
Just added a line for the Amp file to test I'm all set up with my github.

### CODE REVIEW

File: privacy-protections/amp/index.html [hunk 1]
```diff
@@ -40,6 +40,14 @@
       </li>
     </ul>
 
+    <p>URLs containing “amp”</p>
+    <ul>
+      <li>
+        <p><a href="https://www.cultureamp.com/blog/equity-diversity-inclusion-report-2021">Culture Amp</a></p>
```
Comment (SlayterDev) [inline privacy-protections/amp/index.html:46]: Hey @icodebyamanda! This link would probably be better placed under the "False Positives" section: https://github.com/duckduckgo/privacy-test-pages/blob/be687c2db6109c1a8708089cb59751b33a245fc0/privacy-protections/amp/index.html#L71

**Review by SlayterDev — CHANGES_REQUESTED** [review]

File: privacy-protections/amp/index.html (hunk 1 above)
Comment (icodebyamanda) [reply privacy-protections/amp/index.html:46]: @SlayterDev - Makes sense! [I thought this list ](https://app.asana.com/0/0/1201684971137950/f) which I am using to update this file was in sync with it. Thanks!

**Comment by jonathanKingston** [comment]
Hey @icodebyamanda gentle ping on this one :)

**Comment by kdzwinel** [comment]
Closing due to inactivity, feel free to reopen @icodebyamanda when you have time to wrap it up.

**Pull request closed without merging**